src/api/tests/service/billing/test_credit_notifications.py

```
@@ -590,6 +719,71 @@ def test_sync_credit_notification_template_reports_missing_credentials(
+    assert [item["template_code"] for item in payload["items"]] == ["TPL-CACHED"]
+    assert payload["items"][0]["source"] == "local"
+
+
+def test_list_credit_notification_templates_syncs_provider_list(
+    credit_notifications_app: Flask,
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    app = credit_notifications_app
+    app.config.update(
+        ALIBABA_CLOUD_SMS_ACCESS_KEY_ID="key",
+        ALIBABA_CLOUD_SMS_ACCESS_KEY_SECRET="secret",
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "ALIBABA_CLOUD_SMS_ACCESS_KEY_SECRET"

(S106)